test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -249,4 +251,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
   });
+
+  async function initTests() {
+    await kibanaServer.savedObjects.clean({ types: ['visualization'] });
```

Comment:
We really want this method to be sharable across all the visualize tests.  We have almost the same method in the index but I don't think we have an easy way for each test to be able to call it in the index file.  So I think we could put this `init` in the visualize_page and each test could call it from there.